hw/top_earlgrey/doc/_index.md

```diff
@@ -4,78 +4,120 @@ title: "Earl Grey Top Level Specification"
-  - SPI device (using fixed IO) with the following modes of operation: TPM, generic, flash and passthrough
-  - x2 SPI host
-- Other peripherals
-  - Fixed-frequency timer
-  - Always ON (AON) timer
-  - Pulse-Width Modulator (PWM)
-  - Pattern Generator
-- Clock and reset IO and management
-- Software
-  - Boot ROM implementing code secure boot and chip configuration
-  - Bare metal applications and validation tests
-
-## Description
-
-The netlist `top_earlgrey` contains the features listed above, proving basic functionality of the Ibex RISC-V processor core on an FPGA development environment, with a suite of peripherals and memories.
+![Top Level Block Diagram](top_earlgrey_block_diagram.svg)
+
+The OpenTitan Earl Grey chip is a low-power secure microcontroller that is designed for several use cases requiring hardware security.
+The block diagram is shown above and shows the system configuration, including the Ibex processor and all of the memories and comportable IPs.
+
+As can be seen in the blockdiagram, the system is split into a fast processor core domain that runs on a 100MHz jittery clock, and a peripheral domain that runs at 24MHz.
+Further, a portion of the peripheral domain, the analog sensor top and the padring can stay always-on.
+The rest of the system can be shut off as part of the sleep mode.
+
+The OpenTitan Earl Grey chip provides the following features:
+
+<table>
+<thead style='font-size:100%'>
+  <tr>
+    <th colspan="2">OpenTitan Earl Grey Features</th>
+  </tr>
+</thead>
+<tbody style='font-size:90%;line-height:90%'>
+  <tr>
+    <td>
+      <ul>
+        <li>RV32IMCB RISC-V "Ibex" core:
+          <ul>
+            <li>3-stage pipeline, single-cycle multiplier</li>
+            <li>Selected subset of the bit-manipulation extension</li>
+            <li>4kB instruction cache with 2 ways</li>
+            <li>RISC-V compliant JTAG DM (debug module)</li>
+            <li>PLIC (platform level interrupt controller)</li>
+            <li>U/M (user/machine) execution modes </li>
+            <li>Enhanced Physical Memory Protection (ePMP)</li>
+            <li>Security features:
+              <ul>
+                <li>Low-latency memory scrambling on the icache</li>
+                <li>Dual-core lockstep configuration</li>
+                <li>Data independent timing</li>
+                <li>Dummy instruction insertion</li>
+                <li>Bus and register file integrity</li>
+                <li>Hardened PC</li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <br></br>
+        <li>Security peripherals:
+          <ul>
+            <li>AES-128/192/256 with ECB/CBC/CFB/OFB/CTR modes</li>
+            <li>HMAC / SHA2-256</li>
+            <li>KMAC / SHA3-224, 256, 384, 512, [c]SHAKE-128, 256</li>
+            <li>Programmable big number accelerator for RSA and ECC (OTBN)</li>
+            <li>Digital wrapper for analog entropy source</li>
```

Comment:
hmmm... i wonder if we should remove the digital wrapper line.  
Of if we do want to keep it, we should probably mention the compliant health checks (in the documentation we mention both FIPS and CC).